src/combinatorics/simple_graph/matching.lean

```diff
@@ -21,14 +21,13 @@ import combinatorics.simple_graph.basic
 TODO:
   - Lemma stating that the existence of a perfect matching on `G` implies that
     the cardinality of `V` is even (assuming it's finite)
-  - Hall's Marriage Theorem
   - Tutte's Theorem
   - consider coercions instead of type definition for `matching`:
     https://github.com/leanprover-community/mathlib/pull/5156#discussion_r532935457
   - consider expressing `matching_verts` as union:
     https://github.com/leanprover-community/mathlib/pull/5156#discussion_r532906131
 
-TODO: Tutte and Hall require a definition of subgraphs.
+TODO: Tutte requires a definition of subgraphs.
```

Comment:
Please revert the changes in this file; we have a version of Hall's marriage theorem for relations but not for graphs yet.

Reply:
I suppose it may be worth pointing to the relation version of Hall's marriage theorem in the TODO.

Reply:
Ok I reverted and added a reference to combinatorics.hall